editor-integration/emacs/dune-watch.el

```diff
@@ -0,0 +1,196 @@
+;;; dune-watch.el --- provides integration with dune watch  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2021  Kiran Gopinathan
+
+;; Author: Kiran Gopinathan <kirang@comp.nus.edu.sg>
+;; Keywords: 
+
+;; This program is free software; you can redistribute it and/or modify
```

Comment:
Do you mind changing this to MIT to match the rest of dune?

Reply:
Sure, done.